src/forms/progressive-profiling-popup/index.jsx

```diff
@@ -53,6 +53,9 @@ const ProgressiveProfilingForm = () => {
 
   const countryCookieValue = getCountryCookieValue();
   const countryList = useMemo(() => getCountryList(getLocale()), []);
+  const queryParams = useMemo(() => getAllPossibleQueryParams(), []);
+  const showRecommendations = getConfig().ENABLE_POST_REGISTRATION_RECOMMENDATIONS
+    && !('enrollment_action' in queryParams || queryParams?.next);
```

Comment:
wasn't the next query param for embedded experience? is it relevant here? i can't remember exactly but please do confirm this.

Reply:
In authn_mfe we are using the same logic so for feature parity we are using the same logic.